### ISSUE TITLE
input-number 组件, error prop 无效

### ISSUE THREAD

**Issue by ByeBug** [arco-design/arco-design-vue#1000]
- [x] I'm sure this does not appear in [the issue list of the repository](https://github.com/arco-design/arco-design-vue/issues)

## Basic Info 
- **Package Name And Version:** @arco-design/web-vue@2.24.1
- **System:** Windows10
- **Browser:** chrome86.0.4240.198
- **Reproduction Link:** https://codesandbox.io/s/a-input-number-error-test-b66b91

## Extra info
I compared the code of 'input-number' and 'input'.

- input-number:
```js
  setup(props, { emit, slots }) {
    const { size, disabled, error } = toRefs(props);
    // ...
    const { mergedSize: _mergedSize, mergedDisabled } = useFormItem({
      size,
      disabled,
      error,
    });
    const { mergedSize } = useSize(_mergedSize);
```

- input:
```js
  setup(props, { emit, slots, attrs }) {
    const { size, disabled, error, modelValue } = toRefs(props);
    // ...
    const {
      mergedSize: _mergedSize,
      mergedDisabled,
      mergedError: _mergedError,
      feedback,
      eventHandlers,
    } = useFormItem({ size, disabled, error });
    const { mergedSize } = useSize(_mergedSize);
```

Is there a mistake here?

Arco-design is an amazing project, thank you very much!!!  Looking forwar to your reply.

Best wishes.

## What is expected?
The 'error' prop is valid.

**Comment by Flsion** [comment]
@ByeBug Hello~ Welcome to Arco.
There is a problem with the use of the `error` attribute, which should be passed directly to the internal input component. I will fix the problem in the next version

**Comment by flsion** [comment]
fixed at 2.25.0
